### ISSUE TITLE
Support for icons?

### ISSUE THREAD

**Issue by cmmartin** [react-native-community/segmented-control#71]
# Feature Request

Currently the `values` prop accepts an array of strings to display as the label. I'd like to use icons instead. This should be supported on the native side. See the below in Safari -> Bookmarks

<img width="359" alt="Screen Shot 2020-04-18 at 5 23 15 PM" src="https://user-images.githubusercontent.com/2001903/79671587-4cff4a80-8199-11ea-94fe-67afde30eeef.png">

### Code sample

I'm not sure how this would work on the native side yet, but we can add a `renderLabel` prop, so you could do...

```javascript
const renderLabel = value => <Icon name={icons[value]} />

<SegmentedControl values={['foo', 'bar', 'baz']} renderLabel={renderLabel} />
```

Any workarounds or thoughts on the native implementation are appreciated

**Comment by Naturalclar** [comment]
It seems like it is possible to implement image in the UISegmentedControl
based on this SO question https://stackoverflow.com/questions/10480087/uisegmentedcontrol-selected-state-icon-image

currently I don't have much time to look into this, but any PR is appreciated :)

**Comment by Naturalclar** [comment]
Note to myself. 
This PR might help on how to add icon to segmented-control
https://github.com/facebook/react-native/pull/28646

**Comment by tuanmai** [comment]
@Naturalclar Do you have any prefer api for this feature? AFAIK, `UISegmentedControl` support text and image, so it cannot be react component. I need this feature so I'm considering working on it.

**Comment by Naturalclar** [comment]
@tuanmai thanks for the comment! I don't have any preferred api at the moment.

**Comment by Andresain123** [comment]
Hi folks! We are making a new app and we also needed support for images. I’ve just finished a working implementation on the native part. I’m thinking of opening a PR in the next days for supporting images.

![736B676B-8839-4990-A900-4AAA8E9B1E69](https://user-images.githubusercontent.com/4752185/84093372-53bc7800-a9fa-11ea-91e9-093b730a8ac6.jpeg)

Cheers!

**Comment by Naturalclar** [comment]
Just published v2.1.0 with added support for images. Thanks so much @Andresain123 :tada:

**Comment by amirbhz86** [comment]
I created a fork using the react-native-segmented-control module and incorporated support for including both text and images, following this format:

Fork address : https://github.com/amirbhz86/segmented-control

`<SegmentedControl selectedIndex={0} values={[ require('./add.png'), require('./add.png'), require('./add.png)]} iconTexts={["One", "Two", "Three"]} />`
